languages/go/exercises/concept/methods/.docs/introduction.md

```diff
@@ -0,0 +1,61 @@
+## methods
```

Comment:
In this file the same comments apply as in the concept/methods/about.md file

Reply:
Yes